css/style.css

```
@@ -0,0 +1,411 @@
+.wrapper {
+  position: relative;
+}
+.wrapper .slider-main {
+  position: absolute;
+  top: 0rem;
+  bottom: 0rem;
+  left: 0rem;
+  right: 0rem;
+  width: 100%;
+  height: 95.9rem;
+  z-index: 2;
+  background-color: #2c2e3d;
+  max-width: 192rem;
+  margin: 0 auto;
+  position: relative;
+  overflow: hidden;
+  margin-bottom: 20rem;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left {
+  position: absolute;
+  top: calc(50% - 4rem);
+  bottom: unset;
+  left: unset;
+  right: unset;
+  width: 8rem;
+  height: 8rem;
+  z-index: 4;
+  --webkit-display: flex;
+  --moz-display: flex;
+  --ms-display: flex;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  cursor: pointer;
+  color: #ffffff;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
+  background-color: #212331;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left:hover {
+  background-color: #00bcd4;
+  cursor: pointer;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left .arrow__icon {
+  font-size: 4.4444444444rem;
```

gì ấy e, sao để font-size như này

---

1. Em có đặt 1 biến cho Button là 8rem và em muốn là Icon Arrow đó bằng gần 1 nửa của Button nên em chia 1.8 ở bên Style.scss file, render ra Style.css nó ra như thế ah

```suggestion
   $arrow-size: 8rem;
  .arrow__icon {
          font-size: calc($arrow-size / 1.8);
  }
```